src/components/auth/Login.js

```diff
@@ -19,7 +19,7 @@ export const Login = () => {
         Accept: "application/json",
       },
       body: JSON.stringify({
-        email: email.current.value,
+        username: username.current.value,
         password: password.current.value,
       }),
     })
```

Comment:
We should update the login process to set the userId and token (similar to how register currently works): 
localStorage.setItem("rare_user_token", res.token)
localStorage.setItem("rare_user_id", res.user_id)